src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -45,6 +45,30 @@ SELECT * FROM Log
 
 If no data appears after you enable our log management capabilities, follow our [standard log troubleshooting procedures](/docs/logs/log-management/troubleshooting/no-log-data-appears-ui/).
 
+### Openshift requirements [#openshift-req]
+
+To enable this plugin when using OpenShift, additional manual steps are required.
+
+1. Create a `configMap` and RBAC objects before creating the `daemonset`:
+
+   ```shell
+   oc apply -f fluent-conf.yml
+   oc apply -f rbac.yml
+   ```
+
+2. Configure the policy:
+
+   ```shell
+   oc adm policy add-scc-to-user privileged system:serviceaccount:default:newrelic-logging
```

Comment:
Based on what I said about the helm charts, this service account (`default`) seems incorrect.

If you use `nri-bundle`/guided install you may install it in the `newrelic` namespace instead of `newrelic-logging`. The helm chart [creates a daemonset](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/templates/daemonset.yaml#L34) that [uses its own service account](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/templates/serviceaccount.yaml) so `default` does not seem to be the service account that the logging daemonSet is going to use.

Not at least without understanding/explaining what the manifests described above install.